### PR TITLE
Fix Case Issues in Titles of All Learn Pages

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Fix case issues in titles of all Learn pages in both 1.2 and Swan Lake.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/calling-java-code-from-ballerina.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ In this section, we explained how to use the `bindgen` tool to generate Ballerin
 
 The next sections provide more details on various aspects related to Java interoperability in Ballerina. 
 
-## The `bindgen` Tool
+## The 'bindgen' Tool
```
Comment (Samallama) [inline learn/calling-java-code-from-ballerina.md:311]: Should this be normal quotes ('')? If it's Ballerina code it should be within these `` right?

File: learn/deployment/docker.md [hunk 2]
```diff
@@ -580,15 +580,15 @@ target = "java8"
     > docker rmi cc198c0af86e
     ```
 
-### Using a custom base image to build Ballerina Docker images
+### Using a Custom Base Image to Build Ballerina Docker Images
 
 Ballerina ships a base image (e.g., `ballerina/jre8:v1`) with some security hardening. It is used to build Docker images with the user's application code. However, sometimes, you might need to use your own Docker base image depending on your company policies or any additional requirements. This use case shows how to use a custom Docker base image to build Ballerina Docker images with the application code. 
 
-#### Prerequisites
+#### Setting UP The Prerequisites
```
Comment (Samallama) [inline learn/deployment/docker.md:587]:
```suggestion
#### Setting Up the Prerequisites
```

File: learn/keeping-ballerina-up-to-date.md [hunk 3]
```diff
@@ -198,27 +198,27 @@ Distributions available locally:
 …
 ```
 
-### Pull a specific distribution
+### Pull a Specific Distribution
 
 The `ballerina dist pull <distribution>` command downloads a particular distribution and stores it in your local environment. It also sets the fetched distribution as the active distribution.
 
-#### For jBallerina 1.2.5 and above (for update tool version 0.8.8 and above):
+#### For jBallerina 1.2.5 and Above (For Update Tool Version 0.8.8 and Above):
```
Comment (Samallama) [inline learn/keeping-ballerina-up-to-date.md:205]:
```suggestion
#### For jBallerina 1.2.5 and Above (for Update Tool Version 0.8.8 and Above):
```

File: learn/quick-tour.md [hunk 4]
```diff
@@ -74,7 +74,7 @@ Hello Ballerina!
 
 Alternatively, you can use a Ballerina HTTP client to invoke the service.
 
-## Using a client to interact with a network-accessible service
+## Using a Client to Interact With a Network-Accessible Service
```
Comment (Samallama) [inline learn/quick-tour.md:77]:
```suggestion
## Using a Client to Interact with a Network-Accessible Service
```

File: learn/tools-ides/setting-up-visual-studio-code.md [hunk 5]
```diff
@@ -67,9 +67,11 @@ $ code --install-extension <BALLERINA-EXTENSION-DIRECTORY>
 
 > **Tip:** Ballerina Language Specification supports a set of experimental features such as transactions syntax. In order to be compatible with the experimental features and for supporting language intelligence in VS Code Extension, enable the Allow Experimental option in user settings.
 
-> **Troubleshooting**: If you installed a new Ballerina version recently, you might need to restart the VS Code Editor to pick the new Ballerina version. Herein, If you are using Mac OS, press 'Command+Q' keys to quit the app and reopen it.
+## Troubleshooting
 
-The below sections include information on the various capabilities that are facilitated by the VS Code Ballerina Extension for the development process.
+ If you installed a new Ballerina version recently, you might need to restart the VS Code Editor to pick the new Ballerina version. Herein, If you are using Mac OS, press 'Command+Q' keys to quit the app and reopen it.
```
Comment (Samallama) [inline learn/tools-ides/setting-up-visual-studio-code.md:72]:
```suggestion
 If you installed a new Ballerina version recently, you might need to restart the VS Code Editor to pick the new Ballerina version. If you are using Mac OS, press 'Command+Q' keys to quit the app and reopen it.
```

File: swan-lake/learn/quick-tour.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ redirect_from:
 1. Follow the instructions given on the [Getting Started](/swan-lake/learn/getting-started) page to set it up. 
 1. Follow the instructions given on the [Visual Studio Code Plugin](/swan-lake/learn/tools-ides/vscode-plugin) page or the [IntelliJ IDEA Plugin](/swan-lake/learn/tools-ides/intellij-plugin) page to set up your preferred editor for Ballerina.
 
-## Writing a service, running it, and invoking it
+## Writing a Service, Running it, and Invoking it
```
Comment (Samallama) [inline swan-lake/learn/quick-tour.md:19]:
```suggestion
## Writing a Service, Running It, and Invoking It
```

File: swan-lake/learn/using-the-openapi-tools.md [hunk 7]
```diff
@@ -98,7 +98,7 @@ The service generation process is complete. The following files were created.
 -- hello-client.bal
 -- schema.bal
 ```
-#### Generate OpenAPI contract from Service
+#### Generate OpenAPI Contract from Service
```
Comment (Samallama) [inline swan-lake/learn/using-the-openapi-tools.md:101]:
```suggestion
#### Generate an OpenAPI Contract from a Service
```

File: learn/calling-java-code-from-ballerina.md (hunk 1 above)
Comment (praneesha) [reply learn/calling-java-code-from-ballerina.md:311]: Currently, the TOC plugin doesn't support rendering back ticks(``). That’s why I added normal quotes temporarily until we fix [this](https://github.com/ballerina-platform/ballerina-dev-website/issues/1397).

Comment (Samallama) [reply learn/calling-java-code-from-ballerina.md:311]: Same comment in other places

Comment (praneesha) [reply learn/calling-java-code-from-ballerina.md:311]: Currently, the TOC plugin doesn't support rendering back ticks(``). That’s why I added normal quotes temporarily until we fix this.

**Review by Samallama — APPROVED** [review]

**Pull request merged**